src/actions/data_actions.ts

```diff
@@ -65,6 +66,7 @@ export const splitToColumns: ActionSpec = {
   name: _t("Split text to columns"),
   sequence: 1,
   execute: (env) => env.openSidePanel("SplitToColumns", {}),
+  isVisible: (env) => !env.isSmall,
```

Comment:
Some menus are still visible (but it doesn't do anything)
- Find and replace
- More date formats
- Edit table (context menu)